tests/ui/result_map_unit_fn_non_rustfix.stderr

```diff
@@ -0,0 +1,102 @@
+error: called `map(f)` on an Result value where `f` is a unit function
```

Comment:
i'm usually calling these files `unfixable`, which lets us also call files `fixable`

(I'm only calling files `fixable` if there are very few things in that file, otherwise it's just the "main" file)